### PR TITLE
update readme: vault + sftp

### PR DESCRIPTION
**Pull request by san-nat**
update readme for steps for fixing local vault in case it gets into a bad state. also some clean up around sftp content.

## Changes
a) troubleshooting/re-initializing vault: remove vault volume
b) removes a bad sftp link on "Getting Started" readme
c) removes outdated SFTP setup page. (use
https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/getting_started.md#local-end-to-end-tests
)

### Testing
- [x ] Tested locally?
- [ ] Ran `quickTest all`? n/a
- [ ] Ran `./prime test` against local Docker ReportStream container? n/a
- [ ] Downloaded a file from `http://localhost:7071/api/download`? n/a
- [ ] Added tests? n/a

### Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints? n/a
- Does this PR include changes in authentication and/or authorization of existing endpoints? n/a
- Does this change introduce new dependencies that need vetting? n/a
- Does this change require changes to our infrastructure? n/a
- Does logging contain sensitive data? n/a
- Does this PR include or remove any sensitive information itself? n/a

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [x] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

**Review by MauriceReeves-usds — APPROVED** [review]
lgtm

File: prime-router/docs/getting_started.md [hunk 1]
```diff
@@ -307,6 +306,7 @@ If you would like to start with a fresh Vault database, you can clear the Vault
 cd prime_router
 docker-compose down -v
 rm -rf .vault/env/{key,.env.local}
+docker volume rm prime-router_vault # removes vault volume
```
Comment (carlosfelix2) [inline prime-router/docs/getting_started.md:309]: I just found out that docker-compose down -v does the remove of the volumes, so no need for this extra command.  Might as well just close the PR.

Comment (td-usds) [reply prime-router/docs/getting_started.md:309]: Suggestion to close seconded; plus there's `cleanslate.sh` which prunes docker volumes as well...

Comment (san-nat) [reply prime-router/docs/getting_started.md:309]: closing the PR based on above comments

**Pull request closed without merging**